docs/source/usage.rst

```diff
@@ -1,47 +1,75 @@
+such as S3, and **clone** the notebook to their local system.
 
-Every *save* of a notebook creates an *immutable copy* of the notebook on object storage.
+                                             *store*
 
-To simplify implementation, we currently rely on S3 as the object store, using
+Data scientist saves to Local System ------------------> Remote Data Store (i.e. S3) Workspace
+
+                                             *clone*
+
+User requests a notebook to use locally <-------------- Remote Data Store (i.e. S3) Workspace
+
+
+After some time working with a notebook, the data scientist may want to save or share a polished notebook version with
+others. By **publishing a notebook**, the data scientist can display and share work that others can use at a later time.
+
+How to store and clone versions
+-------------------------------
+
+Bookstore uses automatic notebook version management and specific storage paths when storing a notebook.
+
+Automatic Notebook Version Management
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Every *save* of a notebook creates an *immutable copy* of the notebook on object storage. Initially, Bookstore
+supports S3 for object storage.
+
+To simplify implementation and management of versions, we currently rely on S3 as the object store using
 `versioned buckets <https://docs.aws.amazon.com/AmazonS3/latest/dev/Versioning.html>`_.
 
 Storage Paths
--------------
+~~~~~~~~~~~~~
 
-All notebooks are archived to a single versioned S3 bucket with specific **prefixes** denoting the lifecycle of
-the notebook. For example:
+All notebooks are archived to a single versioned S3 bucket using specific **prefixes** to denote a user's workspace
+and an organization's publication of a user's notebook. This captures the lifecycle of the notebook on storage. To do
+this, bookstore allows users to set workspace and published storage paths. For example:
 
-- ``/workspace`` - where users edit
-- ``/published`` - public notebooks (to an organization)
+- ``/workspace`` - where users edit and store notebooks
+- ``/published`` - notebooks to be shared to an organization
 
-Each notebook path is a namespace that an external service ties into the schedule. We archive off versions,
-keeping the path intact (until a user changes them). For example, the prefixes that could be associated with
-storage types:
+Bookstore archives notebook versions by keeping the path intact (until a user changes them). For example, the prefixes
+that could be associated with storage types:
 
 - Notebook in "draft" form: ``/workspace/kylek/notebooks/mine.ipynb``
 - Most recent published copy of a notebook: ``/published/kylek/notebooks/mine.ipynb``
 
-Scheduled notebooks will also be referred to by the notebook ``key``. In addition, we'll need to be able to surface
-version IDs as well.
+When scheduling execution of notebooks, each notebook path is a namespace that an external service can access. This
```

Comment:
I actually wonder if we should remove this until we support some kind of hook into a scheduling system.

Reply:
We can either 1) remove or 2) place in an information box that says that the current release does not support but we expect to do so in a later release. Let me know which you prefer. I don't have a strong feeling either way.

Reply:
Added as a note.